source/customizing_look_and_feel/04_configuring_data_visualizations/data_visualizations.rst

```diff
@@ -116,11 +144,11 @@ To add a custom basemap to the domain:
       * 21
 
 5. In the Label column, indicate a label for the newly added basemap. There must be one label per language (if the domain supports more than one).
-6. Click on the "Save" button in the top right corner of the page.
+6. Click on the Save button in the top right corner of the page.
 
 
 Adding external basemaps
-------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^
 
 Opendatasoft allows the addition of external basemaps through specific integrations.
```

Comment:
The 2 sentences are weird to read, we could explain that we support Mapbox basemaps integration instead of them ?